beacon-chain/light-client-network.md

```diff
@@ -168,9 +176,32 @@ content                              = ForkDigest || SSZ.serialize(light_client_
 content_key                          = selector + SSZ.serialize(light_client_optimistic_update_key)
 ```
 
-> A `None` in the content key is equivalent to the request for the latest
+> A `0` in the content key is equivalent to the request for the latest
 LightClientOptimisticUpdate that the requested node has available.
 
+#### HistoricalSummaries
+
+Latest HistoricalSummaries are stored in the network every 256 epochs (8192 slots).
```

Comment:
```suggestion
Latest `HistoricalSummaries` object is stored in the network every 256 epochs (8192 slots).
```

I think this is what you meant, just adjusting the text a bit to make it clear that a node only is required to store 1.